packages/wired-button/wired-button.js

```diff
@@ -80,12 +85,16 @@ export class WiredButton extends LitElement {
       }
     </style>
     <slot></slot>
-    <div class="overlay">
+    <div class="overlay" tabIndex="${this._getIndex()}" role="button">
       <svg id="svg"></svg>
```

Comment:
Somehow I can't manage to make it work if I set the tabIndex on the shadow host. I was reading about it here and it should work without problems: https://developers.google.com/web/fundamentals/accessibility/focus/using-tabindex but I can't manage to set the focus on the wired-button when using the tab key. I'll look a bit more into this. For now I changed the code in the `_didRender` method and I've set it on `this`. But if you try out the `example.html` file it's not really working when using tab.

Reply:
ok, I asked a colleague and it comes from setting `delegateFocus` to `true`. In the button component there are no focusable elements, so it won't get the focus when setting `tabIndex` on the element itself. @pshihn is there a need for setting it to `true`? Otherwise I'll set it to `false`.

Reply:
Seems like it doesn't work for me even when `delegateFocus` is `false`.

Reply:
Figured it out. you need to set the `delegateFocus` to `false`
and when setting the tabIndex, get the default value from the attribute rather than proeprty
```
this.tabIndex = this.disabled ? -1 : (this.getAttribute('tabindex') || 0);
```